controller/refugee_controller.py

```diff
@@ -21,31 +22,42 @@ def create_refugee(firstname: str,
                    starting_date: date,
                    medical_condition_type: Refugee.MedicalCondition) -> Refugee:
     """
-    Yingbo, Michelle
-    A function used by admin and volunteer.
-    """
-    pass
+    Function to create and save refugee data.
+    """
+    try:
+        for plan in Plan.all():
+            if camp in plan.camps:
+                new_refugee = Refugee(firstname = firstname,
+                                      lastname = lastname,
+                                      num_of_family_member = num_of_family_member,
+                                      starting_date = starting_date,
+                                      medical_condition_type = medical_condition_type)
+                camp.refugees.add(new_refugee)
+                return new_refugee
+    except (Refugee.InvalidNameException,
+            Refugee.InvalidNumOfFamilyMemberException,
+            Refugee.InvalidStartingDateException,
+            Refugee.InvalidCampException) as e:
+        raise ControllerError(str(e))
 
 
 def find_refugee(refugee_id: int) -> Refugee:
     """
-    Yingbo, Michelle
-
+    Function to find if refugee exists and returns the refugee class profile. Raise error if no such refugee exist.
     """
-    pass
+    try:
+        for plan in Plan.all():
+            for camp in plan.camps:
+                refugee = camp.refugees.get(refugee_id)
+                if refugee:
+                    return refugee
```

Comment:
I think you should raise ControllerError if no refugee can be found

Reply:
fixed!